datarobot_provider/operators/monitoring.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2023 DataRobot, Inc. and its affiliates.
+#
+# All rights reserved.
+#
+# This is proprietary source code of DataRobot, Inc. and its affiliates.
+#
+# Released under the terms of DataRobot Tool and Utility Agreement.
+from typing import Any
+from typing import Dict
+from typing import Iterable
+from typing import List
+
+import datarobot as dr
+from airflow.exceptions import AirflowException
+from airflow.models import BaseOperator
+
+from datarobot_provider.hooks.datarobot import DataRobotHook
+
+DATETIME_FORMAT = "%Y-%m-%d %H:%M:%s"
+
+
+class GetServiceStatsOperator(BaseOperator):
+    """
+    Gets service stats measurements from a deployment.
+
+    :param deployment_id: DataRobot deployment ID
+    :type deployment_id: str
+    :param datarobot_conn_id: Connection ID, defaults to `datarobot_default`
+    :type datarobot_conn_id: str, optional
+    :return: Service stats for a Deployment
+    :rtype: List[dict]
```

Comment:
Is it possible to show even more of an example of what kind of key/values are in the dict? Or point them to our public API : https://docs.datarobot.com/en/docs/api/reference/public-api/deployments.html#example-responses_29

Reply:
This information will be added in scope of "Update documentation" as the final task of current milestone